src/main/java/school/lemon/changerequest/java/Matrix.java

```diff
@@ -0,0 +1,155 @@
+package school.lemon.changerequest.java;
+
+import java.io.PrintStream;
+
+/**
+ * Created by lera on 11.12.16.
+ */
+public class Matrix {
```

Comment:
The intention of this task was to introduce some class, that would represent useful operations operate Matrices and you've implemented solution to work with java multi-dimensional arrays (that probably do represent Matrix, but are not Matrix itself).

All your method should work with `Matrix` objects, but not with multi-dimensional arrays.

Also, refer to task documentation:
> Create class matrix. It should contains of:
> 
> 2d array of real numbers
> rows count
> columns count